src/vivarium/component.py

```diff
@@ -25,6 +25,7 @@
 
 from vivarium.framework.artifact import ArtifactException
 from vivarium.framework.population import PopulationError
+from vivarium.types import ScalarValue
```

Comment:
I don't understand this but when I put this in the type checking block it would error saying ScalarValue wasn't defined.